internal/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -42,14 +42,14 @@ type CSVUpdater interface {
 	Apply(*olmapiv1alpha1.ClusterServiceVersion) error
 }
 
-type updaterStore struct {
+type UpdaterStore struct {
```

Comment:
The error occurs because of NewUpdaterStore which will return this structure and is used in another place. See https://github.com/operator-framework/operator-sdk/blob/424a61d56000e6e3d91d352faa1bd4f7c814661f/internal/scaffold/olm-catalog/csv.go#L340.

Reply:
After looking at it, exporting it does make sense now. I'm okay with this.

Reply:
@joelanford I understand that your only concern with this PR still this point. However, it is used outside then shows that could not be in lower case at all. 

So, if you agree and it is ok for you too, I think we could move forward this one and get it merged. WDYT?

Reply:
This is being removed in #2250.

Reply:
Good point so, I will ignore this check for we merge this one.

Reply:
H @estroz and @joelanford, 

It was ignored as well. So, I think we could safely move forward here.